contrib/!config/dockerfile/README.org

```diff
@@ -0,0 +1,26 @@
+#+TITLE: Dockerfile contribution layer for Spacemacs
+
+[[file:img/docker.png]]
+
+* Table of Contents                                                   :TOC@4:
+ - [[#description][Description]]
```

Comment:
With the Toc-org package either you have `GitHub Compliant` links, or `Org-mode Compliant` links... What should we do? it's also just a matter of changing 1 line check #1911